### ISSUE TITLE
Search runs only for 1 iteration

### ISSUE THREAD

**Issue by IlyaTrofimov** [khanrc/pt.darts#22]
Hi!

I run your code with
`python search.py --name cifar10 --dataset cifar10 --batch_size 16 --gpu 2`

and progress doesn't go further this line for several hours
09/10 11:26:50 AM | Train: [ 1/50] Step 000/781 Loss 2.312 Prec@(1,5) (15.6%, 56.2%)

Can you please check where could be a problem?

**Comment by Arui1** [comment]
hi,I have met this problem too. Can you solve it?

**Comment by IlyaTrofimov** [comment]
> hi,I have met this problem too. Can you solve it?

I loaded docker with pt.darts and studied versions of all libraries.
Then I installed same versions via conda, it helped.
I assume some problem with data loader or torchvision version.

**Comment by Arui1** [comment]
thanks, this problem is due to data loader, it stuck.

**Comment by suryakumara** [comment]
> thanks, this problem is due to data loader, it stuck.

Hi can you explain more about data loader problem.. and how to fix it ?

**Comment by Wanli-Jiang** [comment]
I found a very ridiculous solution, you can insert `print(trn_X.shape)` in  https://github.com/khanrc/pt.darts/blob/48e71375c88772daac376829fb4bfebc4fb78144/search.py#L126

**Comment by Wanli-Jiang** [comment]
Try  to set num_worker=0 in dataloader.

**Comment by shield-coder** [comment]
> Hi!
> 
> I run your code with
> `python search.py --name cifar10 --dataset cifar10 --batch_size 16 --gpu 2`
> 
> and progress doesn't go further this line for several hours
> 09/10 11:26:50 AM | Train: [ 1/50] Step 000/781 Loss 2.312 Prec@(1,5) (15.6%, 56.2%)
> 
> Can you please check where could be a problem?

I used pip install torchvision==0.2.1 before loading dataset each time （I use 2080，CUDA10.1，pytorch 1.3），it worked. You should run the codes after you use pip (I think it can be seen as a updating) although it is not a fundamental solution

**Comment by shield-coder** [comment]
pin_memory=False
